address.go

```diff
@@ -134,6 +135,40 @@ func (c *Client) ListAddressesWithContext(ctx context.Context, opts *ListOptions
 	return
 }
 
+// GetNextAddressPage returns the next page of addresses
+func (c *Client) GetNextAddressPage(collection *ListAddressResult) (out *ListAddressResult, err error) {
+	return c.GetNextPageWithContext(context.Background(), collection)
+}
+
+// GetNextAddressPageWithPageSize returns the next page of addresses with a specific page size
+func (c *Client) GetNextAddressPageWithPageSize(collection *ListAddressResult, pageSize int) (out *ListAddressResult, err error) {
+	return c.GetNextPageWithContextWithPageSize(context.Background(), collection, pageSize)
+}
+
+// GetNextPageWithContext performs the same operation as GetNextAddressPage, but
+// allows specifying a context that can interrupt the request.
+func (c *Client) GetNextPageWithContext(ctx context.Context, collection *ListAddressResult) (out *ListAddressResult, err error) {
+	if collection.Addresses == nil || len(collection.Addresses) == 0 {
```

Comment:
Nit: We could check if `has_more` is `false` upfront here, so you don't need to pass in `collection.HasMore` in the `nextPageParameters` function. If you decide to go this way, `nextPageParameters` and `nextPageParametersWithPageSize` can just be combined into one function, you can pass in `nil` if there is no `pageSize` from the parameter.

Reply:
You can't pass nil to an int variable

Reply:
Wait, duh, default 0...

Reply:
If we set it to 0, that would not grab anything, right?